db/imcat

```diff
@@ -4,7 +4,7 @@ R2PM_GIT "https://github.com/stolk/imcat"
 R2PM_DESC "image cat (png)"
 
 R2PM_INSTALL() {
-	gcc -o imcat imcat.c
+	$(CC) -o imcat imcat.c
```

Comment:
```suggestion
	[ -z "$CC" ] && CC=gcc
	${CC} -o imcat imcat.c
```

Reply:
when $CC is not set, make defaults to cc